<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartGameDialog</class>
 <widget class="QWidget" name="StartGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>186</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="blackLabel">
         <property name="text">
          <string>Black</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="blackHuman">
           <property name="icon">
            <iconset theme="user-identity">
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">blackTypeGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="blackAI">
           <property name="icon">
            <iconset theme="computer">
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">blackTypeGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>blackName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="blackName">
         <property name="text">
          <string notr="true">username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Skill level:</string>
         </property>
         <property name="buddy">
          <cstring>blackSkill</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="blackSkill"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>VS</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="whiteLabel">
         <property name="text">
          <string>White</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="whiteHuman">
           <property name="icon">
            <iconset theme="user-identity">
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">whiteTypeGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="whiteAI">
           <property name="icon">
            <iconset theme="computer">
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">whiteTypeGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>whiteName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="whiteName">
         <property name="text">
          <string notr="true">username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Skill level:</string>
         </property>
         <property name="buddy">
          <cstring>whiteSkill</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="whiteSkill"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>blackHuman</tabstop>
  <tabstop>blackAI</tabstop>
  <tabstop>blackName</tabstop>
  <tabstop>blackSkill</tabstop>
  <tabstop>whiteHuman</tabstop>
  <tabstop>whiteAI</tabstop>
  <tabstop>whiteName</tabstop>
  <tabstop>whiteSkill</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="blackTypeGroup"/>
  <buttongroup name="whiteTypeGroup"/>
 </buttongroups>
</ui>
